editoast/src/models/paced_train.rs

```diff
@@ -181,23 +181,26 @@ impl PacedTrain {
     pub fn iter_occurrences(&self) -> impl Iterator<Item = (TrainId, TrainSchedule)> {
         let mut base_occurrences = self.get_base_occurrences();
 
-        let enabled_modified_exceptions = self
+        let modified_exceptions = self
```

Comment:
I think we don't need this variable anymore since it's just an iterator of `self.exceptions`.